README.rst

```diff
@@ -1,5 +1,5 @@
 ⛔️ DEPRECATION WARNING 
-======================
+=======================
```

Comment:
why did we need this?

Reply:
There was a warning so had to made this change to remove warning